iroh-bytes/src/store/traits.rs

```diff
@@ -216,43 +216,6 @@ impl<W: BaoBatchWriter, F: Fn(u64, usize) -> io::Result<()> + 'static> BaoBatchW
     }
 }
 
-/// A combined batch writer
```

Comment:
Yeah, this is dead since we have a proper batch writer. In some branch I removed this already.